### DISCUSSION TITLE
Frontend Initiatives Meeting - March 2, 2021

### DISCUSSION THREAD

**Discussion by amcgee** [dhis2/notes#243] [Meetings] [answered]
When: March 2, 2021 @ 15:00 CET
Where: Remote

# Agenda

- Updates
  - General
    - Dynamic modules (demo?)
    - Currently GitHub Action jobs are stuck in "queued". We are working on a resolution with GitHub.
  - Working groups
    - Developer portal @deboragaleano 
    - Cypress @HendrikThePendric @Mohammer5 
    - UI & Platform 
      - Tree-shakable lib builds are supported, upgrade now!
      - UI tree-shakable build coming soon
    - App Hub & App Management updates @Birkbjo @mediremi 
- Topics
  - PR previews @paschalidi 
  - When will we be able to deploy independently applications? @paschalidi
  - Accessibility update
  - Feedback about support rotation?  What worked, what needs improvement?
  - Anything else?  @dhis2/front-end

**Comment by varl** [comment, marked as answer]
Meeting minutes PR: https://github.com/dhis2/notes/pull/244

Final: https://github.com/dhis2/notes/blob/master/decisions/2021/03/02-fe-initiatives.md
